app/api/api_v1/routers/cluster.py

```diff
@@ -56,7 +56,7 @@ async def cluster_tree(
     my_file = Path(r'static/cluster/' + filename + ".png")
     myPath = str(Path().resolve()).replace('\\', '/') + '/static/cluster'
     if not os.path.exists(my_file):
-        command = 'C:/Program Files/R/R-4.0.4/bin/Rscript.exe'
+        command = 'C:/Program Files/R/R-4.0.3/bin/Rscript.exe'
```

Comment:
we need to add this line to the configuration file? we change this line all the time